databricks/koalas/tests/test_indexes.py

```diff
@@ -1310,3 +1310,15 @@ def test_index_get_level_number(self):
 
         for lv, output in zip(level_names, outputs):
             self.assertEqual(output, kdf.index._get_level_number(lv))
+
+    def test_abs(self):
+        pdf = pd.DataFrame({"a": [-2, -1, 0, 1]}, index=[-2, -1, 0, 1])
+        pidx = pdf.index
```

Comment:
nit question: Why don't we just create an index directly `pd.index([-2, -1, 0, 1])` ?
(Maybe you wanted to test for `Index` from `DataFrame` would work properly ? 😃 )